example/lib/main.dart

```diff
@@ -1,7 +1,7 @@
 //File only for testing. Later file will be deleted.
```

Comment:
Will not merge this file. We want this project to be a drop-in replacement for the original editor until instructed otherwise, so the example should not differ.